### PR TITLE
replaced code with refactored app (RIP: WrapEth Update (Create Raid App template) #51)

### PR DESCRIPTION
**Pull request by bitbeckers**
Live Demo on Netlify: https://infallible-wiles-fd5269.netlify.app
Live Demo on IPFS: https://wild-sound-3616.on.fleek.co
RIP: https://github.com/raid-guild/RIPs/issues/51

Storybook: `yarn storybook`
App: `yarn start`

* Storybook initial setup
* Chakra UI components
* Contexts for state management
* Polygon support
* Log-out button
* Gnosis Safe App
* Prettier/eslint
* Typescripting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,32 +1,63 @@
 {
-  "name": "zipzap-web",
+  "name": "create-raid-app-poc",
```
Comment (dan13ram) [inline package.json:2]: Package name :)

File: public/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="No fees, no frills eth/weth exchange. Built by Raid Guild."
+      content="Web site created using create-react-app"
```
Comment (dan13ram) [inline public/index.html:10]: Public description of website. important for seo

File: src/App.test.js [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
   expect(linkElement).toBeInTheDocument();
```
Comment (dan13ram) [inline src/App.test.js:7]: I think this test will fail, we could just remove it if not needed

**Review by dan13ram — APPROVED** [review]
love this. excited 🎉

File: src/App.test.js (hunk 3 above)
Comment (bitbeckers) [reply src/App.test.js:7]: removed, and removed script from package.json

File: public/index.html (hunk 2 above)
Comment (bitbeckers) [reply public/index.html:10]: updated

File: package.json (hunk 1 above)
Comment (bitbeckers) [reply package.json:2]: renamed to wrapeth-web

**Pull request merged**